### PR TITLE
Feature/car resource

### PR DESCRIPTION
**Pull request by NFKData**

### CODE REVIEW

**Comment by NFKData** [comment]
[Car Resource, API REST implementation (JAX-RS)](https://trello.com/c/cQ3e5WtC/11-car-resource-api-rest-implementation-jax-rs)

File: CarAppBackend/src/main/java/backend/exception/CarNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package backend.exception;
+
+public class CarNotFoundException extends Exception {
```
Comment (javieraviles) [inline CarAppBackend/src/main/java/backend/exception/CarNotFoundException.java:8]: javadocs missing here

File: CarAppBackend/src/main/java/backend/resource/CarResource.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import backend.entity.Car;
+import backend.exception.CarNotFoundException;
+import backend.service.CarService;
+
+@Path("cars")
+public class CarResource {
```
Comment (javieraviles) [inline CarAppBackend/src/main/java/backend/resource/CarResource.java]: as it's been done for the service, i would create an interface with javadocs and other future annotations

File: CarAppBackend/src/main/java/backend/resource/CarResource.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+	@EJB(name = "carService")
+	private CarService carService;
+
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
```
Comment (javieraviles) [inline CarAppBackend/src/main/java/backend/resource/CarResource.java]: extract consumes/produces at class level

**Review by javieraviles — CHANGES_REQUESTED** [review]
some small fixes required

**Review by javieraviles — APPROVED** [review]

**Pull request merged**